plugins/fluentd_telemetry_plugin/src/streaming_config_parser.py

```diff
@@ -71,7 +72,11 @@ def get_ufm_telemetry_xdr_ports_types(self):
                                      self.UFM_TELEMETRY_ENDPOINT_SECTION,
                                      self.UFM_TELEMETRY_ENDPOINT_SECTION_XDR_PORTS_TYPE,
                                      "legacy;aggregated;plane")
-
+    def get_telemetry_timeout(self):
+        return self.safe_get_int(None,
+                                 self.UFM_TELEMETRY_ENDPOINT_SECTION,
+                                 self.UFM_TELEMETRY_ENDPOINT_SECTION_TIMEOUT,
+                                 '60')
```

Comment:
since you use `safe_get_int`, the default value should also be an integer

Reply:
done